recipes/annembed/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: annembed
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+      - {{ pin_subpackage('annembed', max_pin="x") }}
+  skip: True  # [osx]
+source:
+  url: https://github.com/jianshu93/annembed/archive/v{{ version }}.tar.gz
+  sha256: 7d428dd4ac630de7f6ab75e74f7261f1017eae77065f4e678ddde62771253da4
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - {{ compiler('c') }}
+    - rust >=1.39
+    - make
+    - cmake
+
+test:
+  commands:
+    - annembed -h
+
+about:
+  home: https://github.com/jean-pierreBoth/gsearch
+  license: MIT
+  summary: annembed is an ultra-fast and scalable non-linear dimension reduction algorithm (similar to UMAP or t-SNE) for large-scale biological data
```

Comment:
Do you want to add maintainer ? See https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#extra-section